tools/android/MASTG-TOOL-0111.md

```diff
@@ -0,0 +1,18 @@
+---
+title: Blutter
+platform: android
+source: https://github.com/worawit/blutter
+---
+
+[Blutter](https://github.com/worawit/blutter) is an open-source tool created to support the reverse engineering of Flutter applications by compiling the Dart AOT Runtime. It targets the lib files found in decompiled apks and is compatible with the latest versions of Dart. It makes use of an advanced C++20 formatting library. The tool is compatible with Linux, Windows, and macOS operating systems. Blutter is capable of extracting and analyzing Dart objects and can generate Frida scripts for further analysis. It automatically compiles any required Dart versions that are not already installed.
+
+This tool does require a specific environment to work, which can be found [here](https://github.com/worawit/blutter?tab=readme-ov-file#environment-setup).If you don’t want to setup the environment here is the docker support for the blutter tool.
```

Comment:
Where is the docker support?

Reply:
it's not supported yet: https://github.com/worawit/blutter/pull/50 

pancake has one currently: https://github.com/trufae/blutter-docker

Reply:
There is no official docker support as of now, so I have not added it here. But, docker is necessary to work with this tool as it is difficult to get the right dependency. So @cpholguera should we include these in the guide?

Reply:
Yes, please update the paragraph accordingly (you can include both links)

Reply:
I have added the docker file itself, as it is a short file, and this will prevent any broken links while providing ease of usage for this tool.